topics/persistence.md

```diff
@@ -203,6 +219,22 @@ corruption happened to be in the initial part of the file.
 Log rewriting uses the same copy-on-write trick already in use for
 snapshotting.  This is how it works:
 
+**Redis >= 7.0**
+
+* Redis [forks](http://linux.die.net/man/2/fork), so now we have a child
+and a parent process.
+
+* The child starts writing the new AOF in a temporary file.
+
+* The parent open a new incr AOF file to continue writing. If the rewriting fails finally, the old base and incr files (if we have) plus this newly opened incr file represents a full amount of data, so we are safe.
+  
+* When the child is done rewriting the file, the parent gets a signal,
+and use a temporary manifest to track the newly opened incr file and sub-process generated base files.
+
+* Profit! Now Redis do an atomic exchange to the manifest pointer in the memory, so that the result of this AOF rewrite is visible to Redis.
```

Comment:
Isn't the atomic operation renaming the manifest file, not merely a change of a pointer in memory?

Reply:
I updated the above sentence